src/Standards/Generic/Tests/CodeAnalysis/UnusedFunctionParameterUnitTest.inc

```diff
@@ -77,4 +77,4 @@ function bar($x)
 
 function ($a, $b) {
     return $a * 2;
-}
+};
```

Comment:
Is this change needed?

Reply:
Both this one as well as the one below are closures. If the `;` is missing after the closing brace, it is considered a parse error by PHP.

Reply:
/facepalm I completely missed the lack of function name in there.